scripts/create-pr.py

```diff
@@ -0,0 +1,24 @@
+import os
+from datetime import datetime
+GITHUB_BOT_EMAIL = "github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>"
+
+branch_name = str(hash(datetime.now().timestamp()))
+os.system(f'git checkout -b {branch_name}')
+os.system(f'git add ./default-policies/*')
+os.system(f'GIT_COMMITTER_NAME="PROFILE_UPDATE_BOT" git commit --author="{GITHUB_BOT_EMAIL}" -m "Update Seccomp Default Profiles"')
+os.system(f'git push origin {branch_name}')
```

Comment:
Let's replace these with [`subprocess.run(..., check=True)`](https://docs.python.org/3/library/subprocess.html#subprocess.run). This is the *modern* way. :wink: